test/numeric_test.ml

```diff
@@ -0,0 +1,100 @@
+open! Core
+open! Ctypes
+
+(* Tests for numeric types and operations *)
+
+let%expect_test "integer_operations" =
+  Duckdb.Database.with_path ":memory:" ~f:(fun db ->
+    Duckdb.Connection.with_connection db ~f:(fun conn ->
+      (* No need to set single-threaded mode for this test *)
```

Comment:
I think by default we don't expect to need to enable single-threaded mode for tests. Let's remove these comments, since they're just confusing.